### PR TITLE
Improve style checks and test them on travis

### PR DESCRIPTION
**Pull request by jonc125**
I've not fixed issues in the `datasets` app because there's no point until #193 is merged. This PR should then be updated and any remaining style issues fixed.

https://github.com/ModellingWebLab/project_issues/issues/43

### CODE REVIEW

**Comment by jonc125** [comment]
@d20gdx this should be relatively straightforward to review - it's just adding the style check config and then fixing all the issues it threw up. Do ask if anything is unclear!

File: requirements/test.txt [hunk 1]
```diff
@@ -12,14 +12,20 @@ defusedxml==0.5.0         # via python3-openid, social-auth-core
 dj-database-url==0.4.2
 django-braces==1.11.0
 django-guardian==1.4.9
-django==1.11.20           # via model-mommy
```
Comment (sroderick-g5sro) [inline requirements/test.txt]: we've lost the "via model-mommy" comment. I assume thats irrelevant

File: setup.cfg [hunk 2]
```diff
@@ -10,4 +10,4 @@ lines_after_imports=2
 order_by_type=1
 known_first_party=weblab
 include_trailing_comma=True
-skip=wsgi.py,migrations,__init__.py,node_modules
```
Comment (sroderick-g5sro) [inline setup.cfg]: As an aside - I noticed that there is only a secrets.example.py file in the master not a secrets.py. I am assuming this is to force the installer to create their own copy.

File: setup.cfg [hunk 3]
```diff
@@ -1,13 +1,29 @@
 [flake8]
-exclude=**/migrations/*,wsgi.py,__init__.py,**/node_modules/*
```
Comment (sroderick-g5sro) [inline setup.cfg]: wsgi.py and __init__.py no longer being excluded,  is this correct?

File: setup.cfg (hunk 2 above)
Comment (jonc125) [reply setup.cfg]: That's right. No committing sensitive information to a public repo :)
(`secrets.py` is also in the .gitignore file to help avoid people adding it by accident)

File: requirements/test.txt (hunk 1 above)
Comment (jonc125) [reply requirements/test.txt]: Yeah, that's an artifact of an older pip-compile run I think. It's a direct dependency, and should always have been!

File: setup.cfg (hunk 3 above)
Comment (jonc125) [reply setup.cfg]: It is. I figured they didn't throw any style errors, so it didn't make sense to exclude them from the checks!

**Pull request merged**